cmd/expired-authz-purger/main.go

```diff
@@ -166,9 +140,9 @@ func deleteAuthorization(db *gorp.DbMap, table, id string) error {
 	return nil
 }
 
-func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, yes bool, parallelism int, max int) error {
-	for _, table := range []string{"pendingAuthorizations", "authz"} {
-		err := p.purge(table, yes, purgeBefore, parallelism, max)
+func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, parallelism int, max int) error {
+	for _, table := range []string{"authz", "pendingAuthorizations"} {
```

Comment:
Was there a reason to reorder these tables that would merit a comment?

Reply:
Sure, I'll add a comment. Authz is the bigger table and more urgently in need of purging.